Formula/iredis.rb

```diff
@@ -6,28 +6,13 @@ class Iredis < Formula
   url "https://files.pythonhosted.org/packages/33/ee/758d256fdcc53566f8bc852d48402b7f197a78c29684113180d8d7ce95f6/iredis-1.13.0.tar.gz"
   sha256 "d1e4e7936d0be456f70a39abeb1c97d931f66ccd60e891f4fd796ffb06dfeaf9"
   license "BSD-3-Clause"
+  revision 1
   head "https://github.com/laixintao/iredis.git", branch: "master"
 
-  bottle do
-    rebuild 1
-    sha256 cellar: :any_skip_relocation, arm64_ventura:  "08b14d7b3697d8ccadaa394398a41bbf45cc3a846db3b042582abf6f96476c25"
-    sha256 cellar: :any_skip_relocation, arm64_monterey: "0d3797cbfd969ba259032bb00798871d5d347f5b50552c2e4a24752b66c37339"
-    sha256 cellar: :any_skip_relocation, arm64_big_sur:  "0908cbc29191d9286b8ef1343d812612bbede1d7bd119430bd05e64f94c67365"
-    sha256 cellar: :any_skip_relocation, ventura:        "9132e42b5402b89545ca88510cb0f297eb8ea6fcd45a17c63a2ab7ac3da294a1"
-    sha256 cellar: :any_skip_relocation, monterey:       "08171aa56b87c91090bfd98e0c12daba91a6a3a3c6d95fae7f4bb0e12992f309"
-    sha256 cellar: :any_skip_relocation, big_sur:        "618edcd02fa937f4a584efb0816af2b9debb82f60eaa9d84f4a603064770f658"
-    sha256 cellar: :any_skip_relocation, x86_64_linux:   "4bd527e82ced83c56299206e4ded3d6399c938bc3d0f544ae331f86d9a16af82"
-  end
```

Comment:
@woodruffw @alex can we keep the `bottle` blocks? Removing them messes with `test-bot` working out whether a build/test failure should fail CI.

Reply:
Can do: https://github.com/Homebrew/brew-pip-audit/pull/45